### PR TITLE
new tag and job for blst tests

### PR DESCRIPTION
**Pull request by tarakby**
- revert all cross-blst tests
- add a new tag for cross-blst tests 
- add a new ci job for cross-blst tests using Go v1.16

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -96,3 +96,29 @@ jobs:
         timeout_minutes: 15
         max_attempts: 2
         command: make ci-integration
+
+  blst-test:
```
Comment (huitseeker) [inline .github/workflows/ci.yml:100]: Looks great! Could we introduce a periodic build as a [scheduled event](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events) to get signal?

Comment (tarakby) [reply .github/workflows/ci.yml:100]: From what I read about the schedules, it seems to be possible to schedule the entire workflow, but not a specific job. Did you mean to create a new workflow for the blst tests?

Comment (huitseeker) [reply .github/workflows/ci.yml:100]: I'm actually fine with running this job step as an out-of-matrix job for now, we can actually move on to dealing with it as a separate workflow if we get further issues from CI.

**Review by huitseeker — APPROVED** [review]
Thanks a bunch!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tarakby) [reply .github/workflows/ci.yml:100]: Do you mean to leave the job as it is in the PR for now? 
I'm also looking at how to make the job not required for merging PRs but I don't seem to find the GH settings. Any idea how to set this up?

Comment (huitseeker) [reply .github/workflows/ci.yml]: > Do you mean to leave the job as it is in the PR for now?
Yes

> I'm also looking at how to make the job not required for merging PRs but I don't seem to find the GH settings. Any idea how to set this up?


It's a positive setting in status checks for the repo:
https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/about-protected-branches#require-status-checks-before-merging

**Review by zhangchiqing — APPROVED** [review]

**Review by janezpodhostnik — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tarakby) [reply .github/workflows/ci.yml]: @Kay-Zee I think I would need your help to set this new job to not required to merge PRs 🙏

**Pull request merged**